src/main/kotlin/interpreter/pyobject/PyBytes.kt

```diff
@@ -6,7 +6,7 @@ class PyBytes(val wrapped: ByteArray) : PyObject(PyBytesType) {
     override fun pyStr(): PyString = pyRepr()
 
     override fun pyRepr(): PyString {
-        val inner = PyString("...")
+        val inner = PyString(wrapped.joinToString("") { it.toChar().toString() })
```

Comment:
This could simply be `PyString(String(wrapped))` if I'm not mistaken.

Reply:
No, both statements are wrong. repr() of a bytes encodes unprintable bytes to their hex escape. 

```
>>> b"\x01\x07\x7A"
b'\x01\x07z'
```

PyRepr() is also wrong (`"b'\\x01\\x07z'"`).